### ISSUE TITLE
[Bug]: TypeError: issubclass() arg 1 must be a class

### ISSUE THREAD

**Issue by Mapleshade20** [Bing-su/adetailer#146]
### Describe the bug

Adetailer failed to load.

### Full console logs

```Shell
Python 3.9.16 (main, May 17 2023, 17:49:16) [MSC v.1916 64 bit (AMD64)]
Version: <none>
Commit hash: 70b62e083093dd0c09a5210313ef68a918376e6a
Installing requirements

Launching Web UI with arguments:
No module 'xformers'. Proceeding without it.
Error loading script: !adetailer.py
Traceback (most recent call last):
  File "D:\stable-diffusion-webui\modules\scripts.py", line 263, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "D:\stable-diffusion-webui\modules\script_loading.py", line 10, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "D:\stable-diffusion-webui\extensions\adetailer\scripts\!adetailer.py", line 19, in <module>
    from adetailer import (
  File "D:\stable-diffusion-webui\extensions\adetailer\adetailer\__init__.py", line 2, in <module>
    from .args import AD_ENABLE, ALL_ARGS, ADetailerArgs, EnableChecker
  File "D:\stable-diffusion-webui\extensions\adetailer\adetailer\args.py", line 36, in <module>
    class ADetailerArgs(BaseModel, extra=Extra.forbid):
  File "pydantic\main.py", line 198, in pydantic.main.ModelMetaclass.__new__
  File "pydantic\fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic\fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic\fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic\fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "C:\Users\howar\anaconda3\envs\disco-diffusion\lib\typing.py", line 852, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class

Loading weights [e4a30e4607] from D:\stable-diffusion-webui\models\Stable-diffusion\majicmixRealistic_v6.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 6.5s (import torch: 1.3s, import gradio: 2.2s, import ldm: 0.5s, other imports: 0.8s, load scripts: 1.1s, create ui: 0.3s, gradio launch: 0.1s).
Creating model from config: D:\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying optimization: Doggettx... done.
Textual inversion embeddings loaded(5): bad_pictures, badhandv4, easynegative, negative_hand-neg, ng_deepnegative_v1_75t
Model loaded in 2.9s (load weights from disk: 0.6s, create model: 0.2s, apply weights to model: 0.4s, apply half(): 0.5s, move model to device: 0.4s, load textual inversion embeddings: 0.8s).
```

### List of installed extensions

sd-webui-controlnet, disabled.
adetailer, enabled.

**Comment by Mapleshade20** [comment]
I fixed it by updating pydantic to the latest version. This subclass issue can be seen at their(pydantic) repo fixes.

**Comment by Bing-su** [comment]
Thank you for the report. This issue seems to be fixed in pydantic 1.10.8.
